### PR TITLE
Add support for artifacts in TMT/FMF plans

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for artifacts in TMT/FMF plans

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
https://github.com/sclorg/testing-farm-as-github-action/issues/16

File: README.md [hunk 1]
```diff
@@ -17,19 +17,20 @@ API key to the Testing Farm should be stored in your organization's secrets to s
...
+| `tests_tmt_ref`              | A tmt tests branch which will be used for tests                                               | master                        |
+| `compose`                    | Compose to run tests on. [Available composes.](https://api.dev.testing-farm.io/v0.1/composes) | Fedora |
+| `create_issue_comment`       | If GitHub action will create a github issue comment.                                          | false                         |
+| `pull_request_status_name`   | GitHub pull request status name                                                               | Fedora                        |
+| `env_vars`                   | Environment variables for test env, separated by ;                                            | empty                         |
+| `env_secrets`                | Environment secrets for test env, separated by ;                                              | empty                         |
+| `env_artifacts`              | Environment artifacts for test env, separated by ;                                            | empty                         |
```
Comment (fernflower) [inline README.md:31]: Maybe just `artifacts` or `test_artifacts` are a better fit here in terms of naming?

File: action.yml [hunk 2]
```diff
@@ -47,6 +47,14 @@ inputs:
     description: 'Action will update pull request status. Default: true'
     required: false
     default: 'true'
+  copr:
+    description: 'Name of copr to use for the artifacts'
+    required: false
+    default: 'epel-7-x86_64'
+  env_artifacts:
+    description: 'Environment artifacts for test env, separated by ;'
```
Comment (fernflower) [inline action.yml:55]: As long as only `fedora-copr-build` artifacts are supported, maybe it should be mentioned in the input description?

**Comment by phracek** [comment]
@fernflower I have addressed all issues. Feel free to comment / improve / approved.

As soon as this pull request is merged, I will release a new version.

**Review by fernflower — APPROVED** [review]
lgtm!

**Pull request merged**